packages/lib/podbox-shared/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@pcd/podbox-shared",
+  "private": false,
+  "version": "0.0.1",
+  "license": "GPL-3.0-or-later",
```

Comment:
Does this need to be GPL?  I think that most of our code can default to MIT, unless it depends on ZK proof stuff which is GPL.

Reply:
I have no idea - as far as I'm aware we've always used GPL.

Reply:
Based on discussions with Brian and Ivan, we use GPL where we have to (where there are GPL dependencies) and MIT where we don't.  MIT is more permissive (less copyleft) than GPL.  E.g. @pcd/util is MIT, as is @pcd/pod.  @pcd/gpc is GPL because of its circomlib/snarkjs dependency.  I notice passport-interface is GPL, though I'm not sure if it has to be.  This shared code has minimal dependencies so wonder if it could be MIT-licensed.